docs/carts/cart_interactions/add_packages.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+title: Add a package
+parent: Cart interactions
+nav_order: 4
+---
+
+# Add a package to your cart
+
+This form allows you to add a package to your cart. It will create a cart if one does not exist.
+This will add all of the packages included items to the cart.
+> If any of the items are sold out or the quanity of items being added is greater than the quantity available, the entire package will not be added to the cart.
+
+## Form fields
+
+| Field name                         | Description                                            |
+| ---------------------------------- | ------------------------------------------------------ |
+| `package[id]`                      | The ID of the package to add to the cart.              |
+| `package[items][][id]`             | The ID of the package item for which to add modifiers. |
+| `package[items][][modifier_ids][]` | The ID of the modifier to add to the cart.             |
```

Comment:
I like this way of breaking down the fields Murray, going to make a note to do the same on the other pages!